src/index.html

```diff
@@ -1,13 +1,589 @@
+                  class="product__photo"
+                >
+                <h3 class="product__title">Virtual group tours</h3>
+                <p class="product__description">
+                  Request a guided tour, reserve admission and lead your own small group, or request a live virtual tour led by a Museum guide.
+                </p>
+              </article>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+
+            <h2 class="section-title contacts__title">Contact us</h2>
+            <div class="contacts__wrapper">
+
+              <div class="contacts__contact">
+
+                <div class="contact contacts__contact-info">
+                  <h3 class="contact__name">The Met Fifth Avenue</h3>
+                  <h4 class="contact__title">Phone</h4>
+                  <a href="tel:+1 212-535-7710" class="contact__value">+1 212-535-7710</a>
+
+
+                  <h4 class="contact__title">Address</h4>
+                  <a
+                    href="#"
+                    target="_blank"
+                    class="contact__info text-info"
+                  >
+                    1000 Fifth Avenue
+                    New York, NY 10028
+                  </a>
+
+                </div>
+
+                <div class="contact contacts__contact-info">
+                  <h3 class="contact__name">The Met Cloisters</h3>
+                  <h4 class="contact__title">Phone</h4>
+                  <a
+                    href="tel:+1 212-923-3700"
+                    class="contact__value"
+                  >
+                    +1 212-923-3700
+                  </a>
+
+
+                  <h4 class="contact__title">Address</h4>
+                  <a
+                    href="#"
+                    target="_blank"
+                    class="contact__info text-info"
+                  >
+                    99 Margaret Corbin Drive
+                    Fort Tryon Park
+                    New York, NY 10040
+                  </a>
+
+                </div>
+              </div>
+
+              <div class="contacts__form">
+                <form
+                  class="contact__form"
+                  onsubmit="this.reset(); return false;"
+                >
+                  <input
+                    type="text"
+                    class="contact__form__input"
+                    name="name" placeholder="Your name"
+                    required
+                  >
+                  <input
+                    type="email"
+                    class="contact__form__input"
+                    name="e-mail"
+                    placeholder="E-mail"
+                    required
+                  >
+                  <textarea
+                    class="contact__form__input contact__form__input--big"
+                    name="message"
+                    placeholder="Message"
+                    required
+                  >
+                  </textarea>
```

Comment:
to fix this error
```suggestion
                  ></textarea>
```
![image](https://github.com/mate-academy/layout_landing-page/assets/70009205/ac5b8894-7696-4cd7-a525-ff4d67ee88e6)